docs/en_US/hpo_benchmark.rst

```diff
@@ -8,11 +8,11 @@ Terminology
 ^^^^^^^^^^^
 
 
-* **task**\ : a task can be thought of as (dataset, evaluator). It gives out a dataset containing (train, valid, test), and based on the received predictions, the evaluator evaluates a given metric (e.g., mse for regression, f1 for classification). 
-* **benchmark**\ : a benchmark is a set of tasks, along with other external constraints such as time and resource. 
-* **framework**\ : given a task, a framework conceives answers to the proposed regression or classification problem and produces predictions. Note that the automlbenchmark framework does not pose any restrictions on the hypothesis space of a framework. In our implementation in this folder, each framework is a tuple (tuner, architecture), where architecture provides the hypothesis space (and search space for tuner), and tuner determines the strategy of hyperparameter optimization. 
+* **task**\ : a task can be thought of as a tuple (dataset, metric). It provides train and test datasets to the frameworks. Then, based on the returns predictions on the test set, the task evaluates the metric (e.g., mse for regression, f1 for classification) and reports the score. 
+* **benchmark**\ : a benchmark is a set of tasks, along with other external constraints such as time limits. 
+* **framework**\ : given a task, a framework solves the proposed regression or classification problem using train data and produces predictions on the test set. The automlbenchmark framework does not pose any restrictions on the hypothesis space of a framework. In our implementation, each framework is a tuple (tuner, architecture), where architecture provides the hypothesis space, and tuner optimizes the hyperparameters of the architecture. In our implementation, to solve a task, we let the tuner continuously tune the hyperparameters (by giving it cross-validation score on the train data as feedback) until the time or trial limit is reached. Then, the architecture is retrained on the entire train set using the best set of hyperparameters. 
 * **tuner**\ : a tuner or advisor defined in the hpo folder, or a custom tuner provided by the user. 
```

Comment:
"defined in the hpo folder" -> "provided by NNI". And provide a doc link on "a tuner or advisor"

Reply:
fixed